tests/6.types.c

```diff
@@ -1,7 +1,23 @@
 #include <stdio.h>
+#include <stddef.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+// standard types should be lower-cased
+typedef unsigned short ushort;
+typedef float f32;
+typedef size_t size_t;
+typedef void *null;
```

Comment:
That looks bizarre; `null` is not a `void*` typedef. It is usually a 0 value, not a type.

Imho the line should be removed.

Reply:
Or alternatively, another name should be used. The chance of a conflict with whatever the platform decides to do with a very common name like `null`, is much bigger.

Reply:
how about `typedef void *pointer;` ?

Reply:
I have normalized all the example names to `v_u16`, `v_voidptr`, etc...